packages/fluentui/react-northstar/src/components/Carousel/CarouselNavigationItem.tsx

```diff
@@ -61,6 +64,7 @@ class CarouselNavigationItem extends UIComponent<WithAsProp<CarouselNavigationIt
 
   static slotClassNames: CarouselNavigationItemSlotClassNames = {
     indicator: `${CarouselNavigationItem.className}__indicator`,
+    content: `${CarouselNavigationItem.className}__content`,
```

Comment:
what's this for?

Reply:
Used on line 116 

```
defaultProps: () => ({ as: 'span', styles: styles.content }),
```

The way it was before `content` wasn't getting any style applied